main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"code-review-bot/internal"
+	"code-review-bot/internal/web"
+	"code-review-bot/internal/web/routes"
+	"context"
+	"strings"
+	"time"
```

Comment:
The GitHub API client should handle the case when `user` returns `nil` to avoid potential dereferencing of a nil pointer. This addition can prevent runtime panics if GitHub's API responds unexpectedly or with errors related to user fetching.